src/sentry/static/sentry/app/actionCreators/globalSelection.jsx

```diff
@@ -1,43 +1,133 @@
 /*eslint no-use-before-define: ["error", { "functions": false }]*/
+import {isEqual, isInteger} from 'lodash';
 
-import {isInteger} from 'lodash';
+import {getParams} from 'app/views/organizationEvents/utils/getParams';
+import {getUtcDateString} from 'app/utils/dates';
 import GlobalSelectionActions from 'app/actions/globalSelectionActions';
 import sdk from 'app/utils/sdk';
 
+const isEqualWithEmptyArrays = (newQuery, current) => {
```

Comment:
since the querystring looks like `?project=1&project=5&environment=staging&environment=beta`, I assume it won't work properly when converted to an object since there would be duplicate keys?

Reply:
They get parsed into arrays

Reply:
Ahh, oops